tests/Dockerfile.vic-integration-test-base

```diff
@@ -0,0 +1,30 @@
+# VIC Test Image
```

Comment:
Can't we use one of our existing `Dockerfile`s to build this image?  This `Dockerfile` looks sufficiently similar that we can probably just recycle.  It looks like the only thing we need to add is python pip and `robotframework`.